proto/brpb.proto

```diff
@@ -184,6 +184,41 @@ message BackupRequest {
     CipherInfo cipher_info = 14;
 }
 
+message StreamBackupTaskInfo {
+    // The storage for backup, parsed by BR.
+    StorageBackend storage = 1;
+    // The time range for backing up.
+    uint64 start_ts = 2;
+    uint64 end_ts = 3;
+
+    // Misc meta datas.
+    // The name of the task, also the ID of the task.
+    string name = 4;
+    // The table filter of the task.
+    // Only for displaying the task info.
+    repeated string table_filter = 5;
+
+    // The last timestamp of the task has been updated. 
+    // This is a simple solution for unfrequent config changing: 
+    // When we watched a config change(via polling or etcd watching),
+    // We perform a incremental scan between [last_update_ts, now), 
+    // for filling the diff data during conf changing.
+    // The current implementation scan [0, now) for every task ranges newly added, 
+    // So this field is reserved for future usage.
+    reserved "last_update_ts";
+
+    // The prefix at etcd (or other metadata storage) for finding the ranges of the task.
+    // We use '/tidb/br-stream/ranges/<task-name>/<start-key> -> <end-key>' 
+    // as the default key prefix for the target ranges for the task.
+    // Given the path unchanged, the dynamic config(or, hyperlink?) seems useless.
+    reserved "backup_range_key_prefix";
```

Comment:
why do we even need this config 🤔

Reply:
Just a placeholder for the dead 'ranges' 🤔, maybe just remove it

Reply:
Rename it into "ranges" in memory of the original `ranges` field.